interface/login/login.php

```diff
@@ -1,248 +1,247 @@
-<?php unset($_SESSION['relogin']); ?>
-</td></tr>
-<?php endif; ?>
-
-<tr>
-<td><span class="text"><?php echo xlt('Username:'); ?></span></td>
-<td>
-<input class="entryfield" size="22" name="authUser">
-</td></tr><tr>
-<td><span class="text"><?php echo xlt('Password:'); ?></span></td>
-<td>
-<input class="entryfield" type="password" size="22" name="clearPass">
-</td></tr>
-
-<?php
-if ($GLOBALS['language_menu_login']) {
-if (count($result3) != 1) { ?>
-<tr>
-<td><span class="text"><?php echo xlt('Language'); ?>:</span></td>
-<td>
-<select class="entryfield" name=languageChoice size="1">
-<?php
-        echo "<option selected='selected' value='" . attr($defaultLangID) . "'>" . xlt('Default') . " - " . xlt($defaultLangName) . "</option>\n";
-        foreach ($result3 as $iter) {
-	        if ($GLOBALS['language_menu_showall']) {
-                    if ( !$GLOBALS['allow_debug_language'] && $iter[lang_description] == 'dummy') continue; // skip the dummy language
-                    echo "<option value='".attr($iter['lang_id'])."'>".text($iter['trans_lang_description'])."</option>\n";
-		}
-	        else {
-		    if (in_array($iter[lang_description], $GLOBALS['language_menu_show'])) {
-                        if ( !$GLOBALS['allow_debug_language'] && $iter['lang_description'] == 'dummy') continue; // skip the dummy language
-		        echo "<option value='".attr($iter['lang_id'])."'>" . text($iter['trans_lang_description']) . "</option>\n";
-		    }
-		}
-        }
-?>
-</select>
-</td></tr>
-<?php }} ?>
-
-<tr><td>&nbsp;</td><td>
-<input class="button large" type="submit" onClick="transmit_form()" value="<?php echo xla('Login');?>">
-</td></tr>
-<tr><td colspan='2' class='text' style='color:red'>
-<?php
-$ip=$_SERVER['REMOTE_ADDR'];
-?>
-</div>
-</td></tr>
-</table>
-
-</div>
-<div style="clear: both;"> </div>
-<div class="version">
-<a  href="../../acknowledge_license_cert.html" target="main"><?php echo xlt('Acknowledgments, Licensing and Certification'); ?></a>
-</div>
-</div>
-<div class="demo">
-		<!-- Uncomment this for the OpenEMR demo installation
-		<p><center>login = admin
-		<br>password = pass
-		-->
-</div>
-</td>
-</tr>
-</table>
-</form>
-</center>
+<body class="login">
+    <div class="container">
+        <form method="POST" id="loginForm" action="../main/main_screen.php?auth=login&site=<?php echo attr($_SESSION['site_id']); ?>" target="_top" name="login_form">
+            <div class="row">
+                <div class="col-sm-12">
+                    <div class="">
+                        <img class="img-responsive center-block" src="<?php echo $GLOBALS['webroot']?>/public/images/login-logo.svg" />
+                        <input type='hidden' name='new_login_session_management' value='1' />
+                        <?php
+                        // collect groups
+                        $res = sqlStatement("select distinct name from groups");
+                        $groupTmp = sqlFetchArray($res);
```

Comment:
$groupTmp is not an array. This group code is a bit weird. Probably better off not touching it or else will need to test it etc.